lapis-e2e/testData/multiSegmented/testDatabaseConfig.yaml

```diff
@@ -4,6 +4,7 @@ schema:
   metadata:
     - name: primaryKey
       type: string
+      phyloTreeNodeIdentifier: true
```

Comment:
Do we actually have a tree for this case? 

What happens in that case actually? If you don't have a tree in the data, but you still call that endpoint? Do you get an empty response?

Reply:
SILO throws an error: "MRCA action cannot be called on Column '{}' as it does not have a phylogenetic tree associated with it" - as the unit tests actually never call SILO we don't need to add a tree for the unit tests - I added one for the e2e tests :-)